tools/system_libs.py

```diff
@@ -1185,6 +1185,33 @@ class libstandalonewasm(MuslInternalLibrary):
   cflags = ['-Os']
   src_dir = ['system', 'lib']
 
+  def __init__(self, **kwargs):
+    self.is_mem_grow = kwargs.pop('is_mem_grow')
+    super(libstandalonewasm, self).__init__(**kwargs)
+
+  def get_base_name(self):
+    name = super(libstandalonewasm, self).get_base_name()
+    if self.is_mem_grow:
+      name += '-mem_grow'
```

Comment:
Seems odd to mix hyphen and underscore here.

Reply:
Good point, I'll make it `memgrow` (no underscore).